File: 2000A.cpp
#include <iostream>
#include <string>

int main() {
    int t;
    std::cin >> t;

    while (t != 0) {
        std::string s;
        std::cin >> s;

        std::string s1 = s.substr(2);
        int x = std::stoi(s1);

        if (s[0] == '1' && s[1] == '0' && x > 2) {
            std::cout << "YES" << std::endl;
        } else {
            std::cout << "NO" << std::endl;
        }

        t--;
    }

    return 0;
}

File: 2000A.py
t = int(input())
while t !=0 :
    s = input()
    n = len(s)
    if n>2 :
        s1 = s[2:n]
        x = int(s1)
        if s[0] == '1' and s[1] == "0" and x>2 :
            print("YES")
        else :
            print("NO")
    else :
       print("NO")

    t-=1